chart/README.md

```diff
@@ -0,0 +1,1910 @@
+# carto
+
+[The CARTO Helm chart](https://github.com/CartoDB/carto-selfhosted) deploys CARTO in a self hosted environment.
+
+## TL;DR
+
+```console
+# TODO: Chart repository to be replaced by the Carto team
```

Comment:
Is this still a TODO?

Reply:
Yes, that's something the CARTO team will need to handle